x-pack/plugins/observability_solution/metrics_data_access/public/apps/common_providers.tsx

```diff
@@ -5,25 +5,24 @@
  * 2.0.
  */
 
-import { AppMountParameters, CoreStart } from '@kbn/core/public';
+import { CoreStart } from '@kbn/core/public';
 import React from 'react';
-import { KibanaThemeProvider } from '@kbn/kibana-react-plugin/public';
+import { KibanaRenderContextProvider } from '@kbn/react-kibana-context-render';
 import { useKibanaContextForPluginProvider } from '../hooks/use_kibana';
 
 export interface CoreProvidersProps {
   children?: React.ReactNode;
   core: CoreStart;
-  theme$: AppMountParameters['theme$'];
 }
 
-export const CoreProviders: React.FC<CoreProvidersProps> = ({ children, core, theme$ }) => {
+export const CoreProviders: React.FC<CoreProvidersProps> = ({ children, core }) => {
   const KibanaContextProviderForPlugin = useKibanaContextForPluginProvider(core);
 
   return (
-    <KibanaContextProviderForPlugin services={{ ...core }}>
-      <core.i18n.Context>
```

Comment:
`core.i18n.Context` is removed here because it is built-in to `KibanaRenderContextProvider`. There were other opportunities to remove `i18n.Context` in this PR, but the purpose of the Epic being worked on over this PR is only to achieve more consistency in dark mode.